cli/src/cast.rs

```diff
@@ -1,7 +1,14 @@
+pub mod cmd;
+#[allow(dead_code)]
```

Comment:
The reason i included it, is that, building the `cast` binary gives out an unused code error on the `subscriber()` function. The same does not happen on the building `forge`, since it actually uses it.

Should i just move it into the subscriber() function, then ?